source/uwp/Renderer/lib/AdaptiveTableRenderer.cpp

```diff
@@ -73,19 +73,25 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
 
             double cellSpacingDouble = static_cast<double>(tableConfig.CellSpacing());
 
-            // Set left and top margin for each cell (to avoid double margins). Don't set the margin on topmost
-            // or leftmost cells to avoid creating margin outside the table.
-            winrt::Thickness marginThickness = {cellSpacingDouble, cellSpacingDouble, 0, 0};
-            if (columnNumber == 0)
+            if (cellSpacingDouble != 0)
```

Comment:
We were crashing here, so I added a check to only update Margin if there is a non-zero value.